### PR TITLE
fix dav browser error page not styled

### PR DESCRIPTION
**Pull request by blizzz**
Before:

(e.g. https://my.nxt.cld/remote.php/webdav/Foo/Bar/Foobar.pdf)

Screenshot_20200319_234626

Now:

Screenshot_20200319_234546

@nextcloud/designers please help by polishing the design. I bet you know which classes to use :)

### CODE REVIEW

**Review by skjnldsv — APPROVED** [review]

**Review by skjnldsv — COMMENTED** [review]
I meant to comment, wrong issue.

Can we use the existing core templates?  :)

**Review by rullzer — APPROVED** [review]

**Comment by blizzz** [comment]
> I meant to comment, wrong issue.
> 
> Can we use the existing core templates? :)

Yes, but it also needs to be repaired:

Screenshot_20200320_111425

Depending on the error code we can also deliver the 404, which looks good, but the generic exception one does not.

**Comment by blizzz** [comment]
@skjnldsv pushed logic to use core templates. the exception one is ugly, but the 404 is now nice. good enough for this fix? screenshot updated.

File: apps/dav/templates/exception.php [hunk 1]
```diff
@@ -22,12 +22,12 @@
 
 /** @param $_ array */
 ?>
-<span class="error error-wide">
+<div class="body-login-container update">
```
Comment (skjnldsv) [inline apps/dav/templates/exception.php]: this can go away then, no?

Comment (blizzz) [reply apps/dav/templates/exception.php]: aye, the whole  file 🔥

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**